callouts/python/extproc/l4_example/network_basic/network_service_callout_example.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2024 Google LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+import logging
+from typing import Tuple
+from grpc import ServicerContext
+from extproc.service import command_line_tools
+from extproc.service.network_callout_server import NetworkCalloutServer
+
+class BasicCalloutServer(NetworkCalloutServer):
+  """Example callout server.
+
+  A non-comprehensive set of examples for each of the possible callout actions.
+  """
+
+  def on_read_data(
+      self,
+      data: bytes,
+      end_of_stream: bool,
+      context: ServicerContext,
+  ) -> Tuple[bytes, bool]:
+    """Process data from client to server (read path).
+    
+    Override this method to implement custom processing logic.
```

Comment:
The current doc comment makes sense for the superclass, but here for this example, I think the doc comment for on_read_data should just describe what the example on_read_data does (i.e. just logs a message), and doesn't need to repeat the specification of Args and Returns, unless some linter requires it.

Reply:
good catch, done.